SingularityUI/app/styles/taskSearch.styl

```diff
@@ -0,0 +1,4 @@
+@import colors
+
+.remove-query-param
+    color $red
```

Comment:
![image](https://cloud.githubusercontent.com/assets/224895/12960831/60053138-d00b-11e5-874e-8d9d3056bf24.png)

Please turn on your text editor's "ensure newline at end of file" feature to avoid superfluous diffs in the future. :rocket: